coral-component-table/src/scripts/Table.js

```diff
@@ -583,14 +585,35 @@ class Table extends BaseComponent(HTMLTableElement) {
     }
   }
   
+  /** @private */
+  _isTableSticky(){
+    const table = this;
+    const head = table._elements.head;
+    return head && head.sticky;
+  }
+  
+  /** @private */
+  _getActiveHeader(headerCell) {
+    if(this._isTableSticky() === true) {
```

Comment:
please add space after if and you don't need === true AFAICS

Reply:
removed.